site/content/en/docs/administration/advanced/cvat-architecture.md

```diff
@@ -0,0 +1,36 @@
+| **Analytics**           | Vector                                                                                                                                                                                                                       | Event processing                                                                                               | There are several components that process events (backend, frontend, web UI). All events are sent to a single point - Vector, where they are processed and then redirected to ClickHouse. For more information, see [Analytics](/docs/administration/advanced/analytics/).  |
+|                         | ClickHouse                                                                                                                                                                                                                   | Event database                                                                                                 | Stores events. For more information, see [Analytics](/docs/administration/advanced/analytics/).                                                                                                                                                                             |
+|                         | Grafana                                                                                                                                                                                                                      | Dashboards                                                                                                     | Data based on the web interface. For more information, see [Analytics](/docs/administration/advanced/analytics/).                                                                                                                                                           |
+| **Data storage NFS**    | RVVX access mode storage is required in case of multi-node deployment. Available with different types of storages: <br><br><li> AWS: Amazon Elastic File System (EFS)<li>Azure: Azure Files with NFS <li>GCP: Filestore NFS  | Contains data required for CVAT operation.s                                                                    | It is necessary to have the capability for multiple mounting (across several nodes) in RWX mode. For more information, see [K8 Deployment with Helm](/docs/administration/advanced/k8s_deployment_with_helm/)                                                               |
```

Comment:
```suggestion
| **Data storage NFS**    | RVVX access mode storage is required in case of multi-node deployment. Available with different types of storages: <br><br><li> AWS: Amazon Elastic File System (EFS)<li>Azure: Azure Files with NFS <li>GCP: Filestore NFS  | Contains data required for CVAT operations                                                                    | It is necessary to have the capability for multiple mounting (across several nodes) in RWX mode. For more information, see [K8 Deployment with Helm](/docs/administration/advanced/k8s_deployment_with_helm/)                                                               |
```